test/test_prerun.py

```diff
@@ -11,35 +13,113 @@
 from ansiblelint.constants import INVALID_PREREQUISITES_RC
 from ansiblelint.testing import run_ansible_lint
 
+EXECDIR = os.path.dirname(sys.executable)
+
+Deactivator = Callable[[bool], str]
+DeactivateVenv = Callable[[], Deactivator]
+
+
+@pytest.fixture
+def deactivate_venv(monkeypatch: MonkeyPatch) -> Deactivator:
+    """Deliver a function to deactivate the current venv, if any, if requested, by removing it's bin dir from $PATH."""
+
+    def deactiveator(deactivate: bool) -> str:
+        if deactivate:
+            search_path = os.get_exec_path()
+            try:
+                pos = search_path.index(EXECDIR)
+            except ValueError:
+                return ""
+            del search_path[pos]
+            monkeypatch.setenv("PATH", ";".join(search_path))
```

Comment:
I believe this meant to be
```suggestion
            monkeypatch.setenv("PATH", os.path.sep.join(search_path))
```
*nix separator is `:` normally while windows uses `;`

Reply:
@webknjaz And you know very well that ansible-lint does not run on Windows.

Reply:
And your suggestion is misleading as `os.path.sep` is `/` while `os.pathsep` is `:`.-- not sure if just using the `:` not not just be much easier to read.

A simple dot... totally different outcome.